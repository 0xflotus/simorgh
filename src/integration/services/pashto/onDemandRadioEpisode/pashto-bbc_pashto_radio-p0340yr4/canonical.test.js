/**
 * @service pashto
 * @pathname /pashto/bbc_pashto_radio/p0340yr4
 */

import runCanonicalTests from '../../../../pages/onDemandRadioEpisode/canonicalTests';

describe('Canonical pashto onDemandRadioEpisode', runCanonicalTests);
