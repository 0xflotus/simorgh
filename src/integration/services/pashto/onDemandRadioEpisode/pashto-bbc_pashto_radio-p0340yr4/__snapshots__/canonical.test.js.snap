// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical pashto onDemandRadioEpisode Analytics ATI 1`] = `"<img height=\\"1px\\" width=\\"1px\\" alt=\\"\\" style=\\"position:absolute\\" src=\\"https://logws1363.ati-host.net?s=598343&amp;s2=68&amp;p=pashto.bbc_pashto_radio.w172xjcjkv7gw1w.page&amp;x1=[urn:bbc:pips:p0340yr4]&amp;x2=[responsive]&amp;x3=[news-pashto]&amp;x4=[ps]&amp;x7=[player-episode]&amp;x8=[simorgh-nojs]&amp;x9=[وروستي+خبرونه+-+BBC+News+پښتو]\\"/>"`;

exports[`Canonical pashto onDemandRadioEpisode I can see the brand title 1`] = `"وروستي خبرونه"`;

exports[`Canonical pashto onDemandRadioEpisode I can see the episode summary 1`] = `"د نړۍ وروستي خبرونه"`;

exports[`Canonical pashto onDemandRadioEpisode I can see the episode title 1`] = `"۱۸ می ۲۰۲۰"`;

exports[`Canonical pashto onDemandRadioEpisode Media player embed A11y I can read the media player title 1`] = `"اډیو پلیئر"`;

exports[`Canonical pashto onDemandRadioEpisode Media player embed I can see the media player embed 1`] = `"https://polling.test.bbc.co.uk/ws/av-embeds/media/pashto/bbc_pashto_radio/w172xjcjkv7gw1w/ps?morph_env=live"`;

exports[`Canonical pashto onDemandRadioEpisode a11y Assistive technology reads the brand and episode title as the headline 1`] = `"وروستي خبرونه, ۱۸ می ۲۰۲۰"`;
